src/buffer/out/textBuffer.cpp

```diff
@@ -2924,6 +2920,18 @@ void TextBuffer::ScrollMarks(const int delta)
     _trimMarksOutsideBuffer();
 }
 
+void TextBuffer::AddMark(ScrollMark& m, const bool activeMark)

```

Comment:
API nit: `AddMark(..., true)` and `AddMark(..., false)` are unergonomic. What behavior do they capture? Why is `activeMark` pronounced "Active Mark" but defined as "insert mark at position zero?"